lib/submariner_test/submariner_test.sh

```diff
@@ -73,9 +74,9 @@ function combine_tests_basename() {
         secondary_cl_version=$(oc get managedcluster "$secondary_cluster" --ignore-not-found \
             -o jsonpath='{.metadata.labels.openshiftVersion-major-minor}')
 
-        echo "ACM-${acm_ver}-Submariner-${subm_ver}-${primary_cl_platform}-${primary_cl_version}-${secondary_cl_platform}-${secondary_cl_version}-${globalnet}"
+        echo "ACM-${acm_ver}-Submariner-${subm_ver}-${primary_cl_platform}-${primary_cl_version}-${secondary_cl_platform}-${secondary_cl_version}-${globalnet}-${report_suffix}"
     elif [[ "$type" == "ui" ]]; then
-        echo "ACM-${acm_ver}-Submariner-${subm_ver}-${globalnet}-UI"
+        echo "ACM-${acm_ver}-Submariner-${subm_ver}-${globalnet}-UI-${report_suffix}"
```

Comment:
You need to add the dash back to the "report_suffix" local variable definition instead of the final basedname string.
Because if you will not define the "--report-suffix test" argument, the basename of the report file will end up with "-".

Reply:
Don't I need to define local report_suffix=""?
I case it  will not be defined, so it will be an empty string.

Reply:
When you're defining it as `local report_suffix` it's already counted as empty.